docs/high-performance-order-storage/guide-large-store.md

```diff
@@ -0,0 +1,134 @@
+
+To have a sense of how much the final migration (on production) is going to take, make sure to time how much it takes for this command to run. For example, for our test store of 9 million orders, this took about a week to complete.
+
+For extra peace of mind, you can run  **wp wc cot verify_cot_data --verbose** to check that the data has migrated successfully.
+Once migration is complete on your staging site, and still with synchronization enabled, re-run the tests defined in Phase 1. **Disable the sync, and go through all testing again.**
+
+### Test out 3rd party and external systems
+
+You may have non-PHP systems directly connected to DB that would escape a normal code audit. Verify that if you have systems such as data warehouses, shipping tracking systems, accounting systems, etc., they are not reading the posts tables directly. If they are, then they would need to be updated before you can proceed with the next steps.
+
+**Review: Phase 2 Checklist**
+1. [ ] Make sure code on your staging site is up to date by grabbing the latest versions of all WooCommerce-related plugins you’re running.
+2. [ ] Migrate production database to staging site using CLI.
+3. [ ] Monitor migration time for insights into production migration duration.
+4. [ ] Test Phase 1 flows on the staging site with synchronization on.
+5. [ ] Optionally, verify data migration integrity using _wp wc cot verify_cot_data --verbose._
+6. [ ] Test Phase 1 flows on the staging site with synchronization off.
+7. [ ] Audit third-party systems to ensure they do not directly access posts tables.
+
+## Phase 3: Enabling HPOS on production
+
+If you are happy with testing on your local setup and staging site, it’s time to start working on enabling HPOS in production.
+
+### Enable synchronization, keeping posts authoritative
+
+Just like in the previous phases, you’ll want to enable synchronization while still using the WordPress posts storage as order storage. As usual, do this by going to **WooCommerce > Settings > Advanced > Features** and making sure that **Use the WordPress posts tables** is selected as the order data storage.
+
+
+This will make sure that orders will start appearing in the HPOS tables, and that existing orders will stay up to date.
+
+### Migrate and verify data
+
+Now, start the migration by enabling synchronization by toggling on the “**Enable compatibility mode**” checkbox.
+
+Right after enabling synchronization, you can start migrating past orders from the post tables to the new HPOS tables via CLI using the **wp wc cot sync** command.
+
+In case you see any errors, turning off the sync, or killing the sync CLI job is pretty safe. You can resume once you have resolved the error (or if the error was unrelated to the sync process).
```

Comment:
In case you see any errors, turning off the sync, or killing the sync CLI job is pretty safe.

> If you see any errors, turning off the sync or interrupting the the CLI job are considered safe.